app/assets/stylesheets/shared/footer.scss

```diff
@@ -48,7 +57,7 @@
   }
 
   .copyright {
-    color: $footer-text-color;
+    color: #666;
```

Comment:
Color literals like `#666` should only be used in variable declarations; they should be referred to via variable everywhere else.